packages/restlessness-cli/rln-backend/src/models/Endpoint/index.ts

```diff
@@ -14,41 +16,44 @@ export {
   HttpMethod,
 };
 
-export default class Endpoint {
+export interface EndpointEntry {
   id: string
   route: Route
   method: HttpMethod
-  authorizer: Authorizer
-
-  static async getList(): Promise<Endpoint[]> {
-    const endpoints: JsonEndpoint[] = await JsonEndpoint.getList<JsonEndpoint>();
-    return endpoints.map(endpoint => {
-      const ep = new Endpoint();
-      ep.id = endpoint.id;
-      ep.route = Route.parseFromText(endpoint.route);
-      ep.method = endpoint.method;
-      return ep;
-    });
-  }
+  authorizer: AuthorizerEntry
+}
```

Comment:
We really need an interface? The Endpoint class itself should be the interface for an endpoint entry

Reply:
And these properties should be cut and pasted in the Endpoint class itself